daemon/container_unix.go

```diff
@@ -976,9 +987,9 @@ func (container *Container) connectToNetwork(idOrName string, updateSettings boo
 		if err != nil {
 			return err
 		}
-	}
 
-	container.updateSandboxNetworkSettings(sb)
+		container.updateSandboxNetworkSettings(sb)
```

Comment:
I can see how this change makes the code more readable. 
Is there any other functional reason for moving this inside the if check ?
am fine with it BTW.